Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM node:18.16.0-alpine3.17@sha256:44aaf1ccc80eaed6572a0f2ef7d6b5a2982d54481e4255480041ac92221e2f11
+# syntax=docker/dockerfile:1.4
 
-RUN apk --no-cache add zip
+FROM node:18-alpine@sha256:9036ddb8252ba7089c2c83eb2b0dcaf74ff1069e8ddf86fe2bd6dc5fecc9492d
```

Comment:
I'm pretty hesitant to change this.  As you can see, it breaks some things (like apparently not having `zip` on it).  At the least we should specify the exact version rather than using `18-alpine`, just so it makes it easier to verify the sha256 in the future.